OWNERS

```diff
@@ -5,14 +5,12 @@
 approvers:
 - a-roberts
 - alangreene
+- eddycharly
 - mnuttall
 - skaegi
 - steveodonovan
 - dibbles
+- carolynmabbott
 
 reviewers:
```

Comment:
I think we need to remove this entirely as in the website and cli repos

Reply:
Done